### PR TITLE
Debug demo 20200226

### PR DESCRIPTION
**Pull request by theus77**

### CODE REVIEW

File: Controller/ContentManagement/DataController.php [hunk 1]
```diff
@@ -1494,35 +1494,58 @@ public function editRevisionAction($revisionId, Request $request, LoggerInterfac
...
+    {
+        $dataService->hasCreateRights($contentType);
 
         try {
-            $revision = $dataService->newDocument($contentType, $revision->getOuuid(), $jsonContent);
+            $revision = $dataService->newDocument($contentType, null, $rawData);
 
             return $this->redirectToRoute('revision.edit', [
                 'revisionId' => $revision->getId()
             ]);
```
Comment (coppee) [inline Controller/ContentManagement/DataController.php]: (My bad !) Pull out the "happy" redirect from the try and push her at the end of the method ?

File: Controller/ContentManagement/DataController.php [hunk 2]
```diff
@@ -1494,35 +1494,58 @@ public function editRevisionAction($revisionId, Request $request, LoggerInterfac
...
-
-            return $this->redirectToRoute('data.root', [
-                'name' => $contentType->getName()
-            ]);
         }
+
+        return $this->redirectToRoute('data.root', [
+            'name' => $contentType->getName()
+        ]);
```
Comment (coppee) [inline Controller/ContentManagement/DataController.php]: Put the "unhappy" redirect in the catch ?

File: Controller/ContentManagement/DataController.php (hunk 1 above)
Comment (theus77) [reply Controller/ContentManagement/DataController.php]: Or not as the $revision migth not exist outside the try

File: Controller/ContentManagement/DataController.php (hunk 2 above)
Comment (theus77) [reply Controller/ContentManagement/DataController.php]: Or not as the $revision migth not exist outside the try

**Pull request merged**